lib/evmone/analysis.hpp

```diff
@@ -89,7 +89,7 @@ struct block_info
 {
     /// The total base gas cost of all instructions in the block.
     /// This cannot overflow, see the static_assert() below.
-    int32_t gas_cost = 0;
+    uint32_t gas_cost = 0;
```

Comment:
Why is it unsigned now?

Reply:
The basic block can have the cost of 4G instead of 2G.